scripts/create_release.py

```diff
@@ -50,7 +53,12 @@ def main(args):
     f.write(json.dumps(release_info, indent=2))
     f.write('\n')
 
-  subprocess.check_call(os.path.join(script_dir, 'update_bazel_workspace.sh'), cwd=root_dir)
```

Comment:
this wasn't a list before? How did it ever work?

Reply:
Seems like subprocess.check_call can also accept a string .. I didn't know that.   Seems like it treats it like a single element list.. so you can only call programs without any args.